### ISSUE TITLE
GB tower doesnt work on PK Stadium 1

### ISSUE THREAD

**Issue by AlexN64gamer** [project64/project64#1731]
Hi, Project64 team, its nice to communicate with you again, Ive already posted about this issue some time ago, but its still an issue, im using a very powerful cpu, (I9 9900K) tested in Project64 2.3.2 202 with Angrylion RDP plus r8, Azimers Audio v0.70 WIP 10, N-rage 2.5.2 202 with transfer pak in raw data mode and configured correctly (.gb file and .sav file) and Static Interpreter as the RSP plugin, Ive tried with Gliden64 4.0 too in LLE mode, same result, it loads the gb cartridge into the GB tower, but stuck at the Nintendo boot bios screen, after that, nothing more... in angrylions RDP plugin, multithread rendering is checked and compatibility is "Slow, few glitches" Ive tested also PK Stadium 2, GB tower works in that game, it loads the cartridge and you can get in-game (gb) but I have a constant 50 vi (PAL) with my cpu, so its not a problem of processing power, and the gb game is really unplayable on the tower... audio glitches everywhere, its SUPER slow (have to load from the "cartridge" (.gb file) like, every 2 minutes, so... its still an issue worth posting here, hope we can find a way to fix this for the next release, best regards for you and all Project64 suppourters and community :), dont forget to stay home and safe

**Comment by oddMLan** [comment]
Right click **Pokemon Stadium 1 -> Edit Game settings**
Go to Recompiler -> CPU core style
change from Recompiler to Interpreter, then click Ok.
Run the GB tower again. Does that fix the lag?

**Comment by AlexN64gamer** [comment]
Im gonna test it right now

**Comment by AlexN64gamer** [comment]
Nope, it still stucks here 
![No title23](https://user-images.githubusercontent.com/38527600/80846812-6d9dba80-8c05-11ea-9d88-a6ca346d9834.png)

**Comment by AlexN64gamer** [comment]
![No title24](https://user-images.githubusercontent.com/38527600/80846926-c53c2600-8c05-11ea-9fe3-ea1ffcbb4370.png)

**Comment by IronYoshi64** [comment]
Maybe someone can test this in a newer Dev build? Project64 2.3.2.202 is very outdated now.

**Comment by AlexN64gamer** [comment]
this is the lastest dev build v2.4.0-1159-gbdd1d1e Ill be testing it tomorrow and post the results

**Comment by AlexN64gamer** [comment]
![Notitle1png](https://user-images.githubusercontent.com/38527600/80923006-b4f88800-8d78-11ea-87e8-1763890126b4.png)
![Notitle2](https://user-images.githubusercontent.com/38527600/80923009-b7f37880-8d78-11ea-8a46-443db325ae22.png)
![Notitle3](https://user-images.githubusercontent.com/38527600/80923011-baee6900-8d78-11ea-86b5-3b636c61fb74.png)
Tested in the lastest dev build, I get that windows cmd error when trying to change from recompiler to interpreter, later I could change it, but the result is the same, stuck at the Nintendo boot bios GB tower screen

**Comment by oddMLan** [comment]
Thanks for testing
Try the following then. Keep Recompiler mode on. In game settings, set Overclock modifier to 3. Then try again. Does that fix the lag in PKMN Stadium 2?
I doubt it will get you past the boot screen in the first game though. It will have to be looked at some point.

**Comment by AlexN64gamer** [comment]
You mean counter factor?

**Comment by AlexN64gamer** [comment]
there is also a RSP audio signal option that can be checked but I dont think that has anything to do with the jittering and cuts in audio in GB Tower mode

![Notitle5](https://user-images.githubusercontent.com/38527600/80927909-d2d5e500-8d98-11ea-82b5-3537616dd5d0.png)

**Comment by oddMLan** [comment]
Nah, Overclock modifier is an option only in the newest dev builds. You're using v2.3 which is pretty old.

**Comment by AlexN64gamer** [comment]
Ok. forgot to open the other .exe

**Comment by AlexN64gamer** [comment]
Tested in lastest dev build, Overclock modifier 3 on both recompiler and interpreter modes, with same plugins mentioned above, first PKS doesnt past boot screen, second PKS is still extremely laggy (both audio and cartridge load) tested in Nvidia RTX 2060, Intel I9 9900K, 32gb RAM (PAL Spanish games both, but I guess NTSC ones should behave similar)

**Comment by AlexN64gamer** [comment]
https://www.youtube.com/watch?v=I0avm5JIado some footage on Youtube of what GB tower is capable of doing, for those who think its only a SGB emulator, nope, its got some features that make it worth find a way to run it on Project64

**Comment by AlexN64gamer** [comment]
![Untitled23](https://user-images.githubusercontent.com/38527600/80950675-34746e80-8dee-11ea-8c5e-5ea2bc6f9fcf.png)
Pokemon Stadium 2 (Spain) with Pokémon Rojo (red) (spain) on the GB Tower almost working, it still lags and audio is desync and laggy, but works, Overclock modifier on 3 and cpu core mode: Recompiler tested on: dev build v2.4.0-1159-gbdd1d1e

**Comment by LegendOfDragoon** [comment]
Overclocking in gb tower will just make it run even slower.

**Comment by AlexN64gamer** [comment]
I see you addes the Timing Issue label, which is fine for Pokémon Stadium 2, cause it loads the game boy game but its laggy and barely playable, but in the first game it doesnt even boot, so maybe its a different issue in that game?

**Comment by project64** [comment]
maybe, I need to spend time investigating it better to try and guess, that the OC method gets it working it what make me suggest it it timing related and improving time might fix it automatically, or maybe there is some other issue that needs fixing
